doc/cookbook/db-config.rst

```diff
@@ -149,4 +149,6 @@ Configuring the client in ``pipeline.yaml``
 -------------------------------------------
 
 Check out the :doc:`../get-started/sql-pipeline` to learn how to configure
-the database client in your ``pipeline.yaml`` file.
\ No newline at end of file
+the database client in your ``pipeline.yaml`` file. 
+
+You can see a complete 'example <https://github.com/ploomber/projects/tree/master/cookbook/sql-dump>'_.
```

Comment:
thanks for adding this. i think it'd make sense to create a new section, something like this:

```
SQL Examples
---------------

To see some examples using SQL connections, see this:

[list of relevant links]

```

Reply:
sure will do that

Reply:
here are two other links you can include: https://github.com/ploomber/projects/tree/master/templates/etl

https://github.com/ploomber/projects/tree/master/templates/spec-api-sql

Reply:
also, check the links syntax: 

```
`Link text <link URL>`_
```

Reply:
are the 2 additional links also for the 'Database Configuration' page?

Reply:
yep